articles/web-dev's-guide-SEO/index.md

```diff
@@ -0,0 +1,85 @@
+
+Search engine optimization is the art of standing out amongst other websites. This art is pioneered by creating an experience that the user seeks repetitively, thereby increasing the presence on the internet. We will look at some of the key considerations a developer should keep in mind while designing the website. The techniques aim at simplifying the job of the millions of [crawlers that search and index the internet regularly](https://www.sciencedaily.com/terms/web_crawler.htm). We will look at the various aspects of SEO, and consider the important concepts of Metadata. Towards the end of the article, you will have a solid grasp of some good SEO techniques. 
+
+## Traditional SEO
+
+Let us look at some of the ways that are used to improve SEO. These are factors responsible for ensuring crawlers can do their job easily. Earlier, the HTML meta tag was used to fill in a set of important keywords. Severe exploitation of the same has resulted in search engines disregarding the keywords. Crawlers are bots that go through pages and find the content present there and list the various links the website directs to. By doing so, crawlers map the internet and enhance the performance of search engines. 
+
+At the same time, the content plays a major role. The amount of time a user spends and the monetization capacity are tracked regularly by the crawlers. Finally, target the niche audience matters. Providing unique content that is designed, delivered, and branded has a higher chance of being ranked higher. The [investment on User Interface and User Experience(UI/UX)](https://neilpatel.com/blog/googlebot-optimization/) returns a larger time spent on the site and more number of clicks resulting in better SEO. 
+
+## PageRank
+The [PageRank](https://en.wikipedia.org/wiki/PageRank) algorithm, named after Larry Page, is the algorithm used to measure and rank the importance of websites. The underlying assumption that ranking algorithms make is as follows: the quality of a site is directly proportional to the number of links to it. Quality links are links to pages that are ranked better. 
+
+Now that we have sufficient background about the topic, let us look at some pro-SEO dev techniques to boost the rankings higher and higher.
+
+### Metadata
+
+1. **Adding meta tag**: Adding meta tags is the oldest way of implementing SEO. These meta tags are placed inside the head tag along with other link tags. Let us look at examples for the same.
```

Comment:
Add the between adding and meta.

Change to Let us look at the example below for readability.